database/seed.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const db = require('./connection.js');
+const { categories } = require('./mockhelperdata.js');
+
+db.connect();
+
+
+const dataPath = path.join(__dirname, '/data.csv');
+const seedQuery = `COPY restaurants (name, category_id, delivery_time, favorited, image_url, location) FROM '/mnt/c/code/SDC/nearby-carousel/database/data.csv' DELIMITER ',' CSV HEADER`;

```

Comment:
The absolute path (`/mnt/...`) might cause headaches later. Is there a way to use `__dirname` and specify it relative to the current directory?

Reply:
Absolutely, I was loathe to use template strings due (irrationally) to the injection thing, and they're for sure fine to use here. Thanks!